samples/tacos/src/main/kotlin/com/slack/circuit/tacos/model/Ingredient.kt

```diff
@@ -0,0 +1,18 @@
+package com.slack.circuit.tacos.model
+
+import android.os.Parcelable
+import androidx.compose.runtime.Immutable
+import kotlinx.parcelize.Parcelize
+
+typealias Cents = Int
+
+@Parcelize
```

Comment:
Why does it need to be parcelable?

Reply:
Nope. Removed in [06b1250](https://github.com/slackhq/circuit/pull/496/commits/06b1250f27c1e130a3063695e3ee2086a0bc8ed4)